plugins/modules/win_nssm.ps1

```diff
@@ -477,6 +483,16 @@ if ($state -eq 'absent') {
         Update-NssmServiceParameter -parameter "AppStdout" -value $stdoutFile @common_params
         Update-NssmServiceParameter -parameter "AppStderr" -value $stderrFile @common_params
 
+        # note: convert app_env dictionary to list of strings in the form key=value and pass that a long as value
+        $tmp = $( $app_env.GetEnumerator() | ForEach-Object { "$($_.Name)=$($_.Value)" } )
```

Comment:
I would use a proper name for this var like `$app_env_strings` or something. The subexpression isn't doing anything special here as well so you may as well make it like

```suggestion
        $tmp = $app_env.GetEnumerator() | ForEach-Object { "$($_.Name)=$($_.Value)" }
```

Reply:
Done